README.md

```diff
@@ -80,6 +80,9 @@ If you are new to Diffy, please refer to our [Quickstart](/QUICKSTART.md) guide.
         -proxy.port=:8880 \
         -admin.port=:8881 \
         -http.port=:8888 \
+        -candidateApiRoot='api/v4/?param1=value1&param2=value2' \
+        -primaryApiRoot='api/v3/' \
+        -secondaryApiRoot='api/v3/' \
```

Comment:
Diffy assumes primary and secondary to be 2 instances of the same version of the service. Is there ever a reason for primary and secondary to have different api roots? If not, let's just keep one.